lib/iris/tests/integration/test_grib_load.py

```diff
@@ -83,18 +56,11 @@ def test_load_3_layer(self):
         self.assertCML(cubes, ("grib_load", "3_layer.cml"))
 
     def test_load_masked(self):
-
         gribfile = tests.get_data_path(
             ('GRIB', 'missing_values', 'missing_values.grib2'))
         cubes = iris.load(gribfile)
         self.assertCML(cubes, ('grib_load', 'missing_values_grib2.cml'))
 
-    @skip_irisgrib_fails
-    def test_y_fastest(self):
-        cubes = iris.load(tests.get_data_path(("GRIB", "y_fastest",
-                                               "y_fast.grib2")))
-        self.assertCML(cubes, ("grib_load", "y_fastest.cml"))
-
```

Comment:
@lbdreyer I don't have the context, but why are all of these tests now not relevant?